src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,60 @@
+package duke.exception;
+
+/**
+ * Represents a DukeException class
+ *
+ * @author Khor Jun Wei
+ * @version CS2103T AY22/23 Sem 1
+ */
+public class DukeException extends Exception {
+
+    /**
+     * Represents a message.
+     */
+    private String msg;
+
+    /**
+     * Constructor for DukeException.
+     * @param message message to determine the error
+     */
+    public DukeException(String message) {
+        if (message.equals("todo")) {
```

Comment:
Long chains of if-else isn't very friendly. Perhaps you can consider suing switch-case or use a HashMap?